qcfractal/tests/test_collections.py

```diff
@@ -159,6 +159,35 @@ def test_dataset_compute_response(fractal_compute_server):
     assert len(response.ids) == 2
 
 
+def test_dataset_d3(fractal_compute_server):
+    testing.check_has_module("psi4")
+    testing.check_has_module("dftd3")
+
+    client = ptl.FractalClient(fractal_compute_server)
+    ds_name = "He_DFTD3"
+    ds = ptl.collections.Dataset(ds_name, client)
+
+    # Add two helium dimers to the DB at 4 and 8 bohr
+    HeDimer = ptl.Molecule.from_data([[2, 0.343, 0, -4.123], [2, 0, 0, 4.123]], dtype="numpy", units="bohr", frags=[1])
+    ds.add_entry("HeDimer", HeDimer)
+    ds.set_default_program("psi4")
+    ds.add_keywords("scf_default", "psi4", ptl.models.KeywordSet(values={}), default=True)
+
+    ds.save()
+
+    ds.compute("B3LYP", "sto-3g")
+    ds.compute("B3LYP-D3", "sto-3g")
```

Comment:
I don't see any new tests that are not in `test_compute_reactiondataset_dftd3`?

Reply:
This test works on Dataset instead of ReactionDataset. The bug was present in Dataset only, so the previous test did not catch it.